shared/tasks.ts

```diff
@@ -0,0 +1,16 @@
+import {PubSub} from '@google-cloud/pubsub';
+
+import {setStatusOnGithub} from './github';
+
+const pubsub = new PubSub();
+
+/** 
+ * Creates a task to be run as soon as quota is check and update the status
+ * if the PR can be cherry-picked.
+ */
+export async function createPresubmitPrTask(
+    org: string, repo: string, pr: string, branch: string, sha: string) {
+  await setStatusOnGithub(org, repo, sha, 'pending');
+  const data = JSON.stringify({org, repo, pr, branch, sha});
+  await pubsub.topic('presubmit-pr-task').publisher().publish(data);
```

Comment:
Can we store the `topic(XXX).publisher` as a constant, so that we don't need to re-create it all the time?